### ISSUE TITLE
breaking change in rotary-embedding-torch dep

### ISSUE THREAD

**Issue by martinshkreli** [neonbjb/tortoise-tts#705]
lucidrains removed broadcat method, which tortoise depends on:

def broadcat(tensors, dim = -1):
    broadcasted_tensors = broadcast_tensors(*tensors)
    return torch.cat(broadcasted_tensors, dim = dim)

https://github.com/lucidrains/rotary-embedding-torch/commit/02a5d0747249183789eef722ecbcd637a5dce88d

**Comment by rsxdalv** [comment]
quickfix is to pin rotary_embedding_torch==0.4.0
also made an issue clarifying if this API change was intentional as it might affect other downstream projects https://github.com/lucidrains/rotary-embedding-torch/issues/17
